src/strategies/delegation/index.ts

```diff
@@ -39,7 +39,7 @@ export async function strategy(
     addresses.map((address) => {
       const addressScore = delegations[address]
         ? delegations[address].reduce(
-            (a, b) => a + scores.reduce((x, y) => x + y[b] || 0, 0),
+            (a, b) => a + scores.reduce((x, y) => y[b] ? (x + y[b]) : (x + 0), 0),
```

Comment:
Why x + 0 and not x directly?

Reply:
Oops! Yes, how did i miss that, changing